src/css/components/_results.scss

```diff
@@ -15,4 +15,15 @@
     min-width: 11.3em;
     border-radius: 2px;
   }
+
+  .saving {
+    color: green;
+  }
+  .increase {
```

Comment:
i think this guy could me named as `increasing` just to be more concise with `saving`, or even change `saving` to `saved` and `increase` to `increased`.

Reply:
I decided to change to more neutral names "success / error".

Reply:
good point, success in reducing size, but error is not a really error. 🤔